src/core/Akka.Streams/Implementation/Fusing/GraphInterpreter.cs

```diff
@@ -270,7 +270,7 @@ public sealed class Connection
         /// <summary>
         /// TBD
         /// </summary>
-        public static readonly bool IsDebug = false;
+        public const bool IsDebug = false;
```

Comment:
Putting this as **const** is actually important. Otherwise compiler won't optimize away all branches in akka.streams interpreter hot path.